### DISCUSSION TITLE
Generation Token API

### DISCUSSION THREAD

**Discussion by crapulas-cb6** [bokub/conso-api#61] [Aide]
Bonjour,

Lorsque je suis le lien pour générer mon token, j'arrive bien sur une page "PArtage de vos données d'électricité"
Je vois la ref de mon compteur dans le cadre Point de livraison concerné par le partage mais en passant la souris dessus, il indique "compteur incompatible / Il doit collecter les données listées ci-dessus"

Que faire?

Merci

**Comment by bokub** [comment]
Salut,

Tu peux m'envoyer une capture d'écran ?

Merci

**Comment by crapulas-cb6** [comment]
![image](https://github.com/user-attachments/assets/644ed777-3e4f-4af9-bb08-e8b71ba268dc)

**Reply by bokub** [reply]
En effet, le problème a été remonté dans #62

**Comment by 21042012** [comment]
Bonjour,

J'ai exactement le même souci 
Si je fais une capture ce sera égale à celle ci-dessus

Merci par avance

**Reply by bokub** [reply]
Probleme déjà remonté dans #62
